flake8_trio/visitors/visitor100.py

```diff
@@ -5,8 +5,9 @@
 the timeout can only be triggered by a checkpoint.
 Checkpoints on Await, Async For and Async With
 """
-# if future annotations are imported then shed will reformat away the Union use
-from typing import Any, Union
+from __future__ import annotations
+
+from typing import Any
```

Comment:
not using decorators -> don't have to care about libcst's type checking not handling `|`